frontends/qt/main.cpp

```diff
@@ -109,13 +109,12 @@ class WebEngineView : public QWebEngineView {
 
 int main(int argc, char *argv[])
 {
-    // note: doesn't work as expected. Users with hidpi enabled should set the environment flag themselves
-    // turn on the DPI support**
-// #if QT_VERSION >= QT_VERSION_CHECK(5,6,0)
-//     QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
-// #else
-//     qputenv("QT_AUTO_SCREEN_SCALE_FACTOR", QByteArray("1"));
-// #endif // QT_VERSION
+// Enable auto HiDPI scaling on Windows only for now.
+// Historically, auto scaling did not work as expected on other platforms
+// before Qt v5.14 but we're at 5.11 due to older systems support.
+#if defined(_WIN32) && QT_VERSION >= QT_VERSION_CHECK(5,6,0)
```

Comment:
since we are at least on qt 5.11, the `QT_VERSION >= QT_VERSION_CHECK(5,6,0)` check could be skipped.

Reply:
True but so could the others below.
How about I make a separate PR to remove all these conditions and update readme in places where it mentions Qt dependencies?